templates/head-meta.html

```diff
@@ -10,7 +10,7 @@
 
 <link rel="publisher" href="https://plus.google.com/107187849809354688692">
 <link rel="shortcut icon" href="/images/logos/p-logo-32.png">
-<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
+<link rel="preload" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
```

Comment:
Yeah it got a commit that deletes it (will deal without preload in a separate PR) but I'm stuck fixing  bad FOUC the shell.html introduces so that's why it's not up yet.